lib/setup.js

```diff
@@ -102,13 +106,43 @@ function getInput(name, pattern, defaultValue) {
   return value;
 }
 
+/**
+ * Installs SAM CLI using the native installers.
+ *
+ * See https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html
+ *
+ * @param {string} version - The SAM CLI version to install.
+ * @returns {Promise<string>} The directory SAM CLI is installed in.
+ */
+// TODO: Support more platforms
+// TODO: Support caching
+async function installUsingNativeInstaller(version) {
+  if (os.platform() !== "linux" || os.arch() !== "x64") {
+    core.setFailed("Only Linux x86-64 is supported with installer: true");
+    return "";
+  }
+
+  const url = version
+    ? `https://github.com/aws/aws-sam-cli/releases/download/v${version}/aws-sam-cli-linux-x86_64.zip`
+    : "https://github.com/aws/aws-sam-cli/releases/latest/download/aws-sam-cli-linux-x86_64.zip";
```

Comment:
Weird that it's not `/releases/download/latest/`; oh well.

Reply:
🤷